src/github/client/interceptors.ts

```diff
@@ -75,6 +77,12 @@ export const instrumentFailedRequest = (metricName) =>
 export const handleFailedRequest = (logger: Logger) =>
 	(error) => {
 		const response = error.response as AxiosResponse;
+
+		if (response?.status === 408 || error.code === "ECONNABORTED") {
```

Comment:
That's the only way to determine if it is a timeout with Axios client (See https://github.com/axios/axios/issues/1174)

Reply:
Yes, but no.  This can also be the status/error code for aborting a request from our client side.

You should use the axios specific timeout code of `ETIMEDOUT`.  Upgrade Axios to the latest version (0.25.0) and add a transitional object to the config:
```
transitional: {
    clarifyTimeoutError: true
  },
  ```

This will make axios differentiate between the two.  I believe this will be the default moving forward.

Reply:
More on this here: https://github.com/axios/axios/blob/master/lib/adapters/xhr.js#L133

There's a lot of talk about this on the project and by using this pattern will help future proof our code for new axios versions.

Reply:
Oh great, I thought they didn't implement it.